deploy/olm-catalog/open-liberty-operator-community/0.5.1/open-liberty-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,396 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The file name should be `open-liberty-operator.v0.5.1.clusterserviceversion.yaml`